### ISSUE TITLE
Photon and environment variables spec

### ISSUE THREAD

**Issue by divyenduz** [prisma/specs#343]
https://github.com/prisma/prisma2/issues/956

Not shipping Photon with `dotenv` or a similar library to support `.env` files without config was a conscious decision. 

This issue tracks speccing this behavior in the Photon spec.

**Comment by matthewmueller** [comment]
@divyenduz can you share why we don't allow runtime environment variables to be set? Is it because the query engine needs to load a static schema?

**Comment by divyenduz** [comment]
@matthewmueller We do allow runtime environment variables, the user needs to load them via `dotenv.config()` or something similar because we didn't want to ship `dotenv` with Photon.

**Comment by matthewmueller** [comment]
Oh, maybe I misread, so this is a bug and we should fix it, right?

```js
import { Photon } from '@generated/photon'
import * as dotenv from 'dotenv';

dotenv.config(); // Load the environment variables
const photon = new Photon()
```

**Comment by divyenduz** [comment]
Yes, if this doesn't work, then it is a bug and should be fixed.

**Comment by matthewmueller** [comment]
Oh my bad, I didn't see.

>  But, when we use photon js, it won't pick it up. To fix this, we have to use dotenv right before instantiating photon like this.

This user was looking for automatic loading of `.env`, which we won't support. Thanks for clarifying!
